RHEL6_7/selinux/RestoreConfig/postupgrade.d/restoreConfig.sh

```diff
@@ -2,7 +2,25 @@
 
 SEMANAGE_EXPORT_FILE=semanage_export
 
+semanage_syntax_update() {
+
+    ftype_opts=( '-f' '-ftype')
+    bool_opts=( '-1' '-0' '--on' '--off' )
+
+    for opt in ${bool_opts[@]}
+    do
+        sed -i -r 's/'"$opt"'/-m '"$opt"'/g' "$1"
+    done
+
+    for opt in ${ftype_opts[@]}
+    do
+        sed -i -r 's/'"$ftype_opt"'\sdirectory/'"$ftype_opt"' d/g' "$1"
```

Comment:
`$ftype_opt` doesn't exist. Probably you meant to use `$opt` or change name on the 15 line.

Reply:
Why `\s`? Is the space guarranteed to be single byte? If so, isn't also type of space guarranteed?  Then you could use just that space and avoid `-r` and also the quoting dance. Just sayin... :)